### PR TITLE
Added user-friendly 'clear ip arp' error messages

### PR DESCRIPTION
**Pull request by obaidamin**

### CODE REVIEW

File: src/CLI/actioner/sonic-cli-neighbors.py [hunk 1]
```diff
@@ -211,8 +211,12 @@ def run(func, args):
         elif 'sonic-neighbor:output' in response.keys():
             status = response['sonic-neighbor:output']
             status = status['response']
-            if (status != "Success"):
+            if "255" in status:
+                print "Unable to clear all entries, please try again"
```
Comment (venkatmahalingam) [inline src/CLI/actioner/sonic-cli-neighbors.py:215]: As discussed, it is expected for Linux to return error when there are lot of ARP entries..please try atleast 3 times and send error from backend.

File: src/CLI/actioner/sonic-cli-neighbors.py [hunk 2]
```diff
@@ -211,8 +211,12 @@ def run(func, args):
         elif 'sonic-neighbor:output' in response.keys():
             status = response['sonic-neighbor:output']
             status = status['response']
-            if (status != "Success"):
+            if "255" in status:
```
Comment (venkatmahalingam) [inline src/CLI/actioner/sonic-cli-neighbors.py:214]: We need the below message for REST intf as well, please add appropriate error/warn string in the RPC handler.

Comment (venkatmahalingam) [inline src/CLI/actioner/sonic-cli-neighbors.py:214]: What's 255 in status, did we mention anything about 255 in YANG, please fix it appropriately.

Comment (obaidamin) [reply src/CLI/actioner/sonic-cli-neighbors.py:214]: I know, this is a temporary fix as any major change will not be merged right now. Post Buzznik there will be a major update that will address all these issues.

**Review by venkatmahalingam — APPROVED** [review]

**Pull request closed without merging**